aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -406,6 +408,17 @@ public void onLoadBeginSnapshot(
         }
     }
 
+    public ControlledFragmentHandler.Action onUnknownMessageSchema(
+        final int schemaId,
+        final int templateId,
+        final DirectBuffer buffer,
+        final int offset,
+        final int length,
+        final Header header)
+    {
+        return consensusModuleExtension.onFragment(schemaId, templateId, buffer, offset, length, header);
```

Comment:
Fragments would be assembled at this stage so it would be `onMessage`.

Reply:
renamed